src/interfaces/GemLike.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-FileCopyrightText: © 2017, 2018, 2019 dbrock, rain, mrchico
+// SPDX-FileCopyrightText: © 2023 SDAO Foundation <www.sdaofoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity >=0.8.8;
+
+interface GemLike {
```

Comment:
For this and the Vest interfaces you can use `dss-interfaces`: https://github.com/makerdao/dss-interfaces/blob/master/src/ERC/GemAbstract.sol and https://github.com/makerdao/dss-interfaces/blob/master/src/dss/VestAbstract.sol respectively.

I don't mind too much having the interfaces in here, but using `dss-interfaces` saves you some time. It's what it's there for. :)

Reply:
Since it's only a couple of small interfaces, I rather have them inlined here than introduce a new dependency.

Reply:
`dss-interfaces` is actually already a dependancy of `dss-test` FYI, but I don't care much about this either way.